packages/core/lib/commands/console/meta.js

```diff
@@ -27,6 +43,8 @@ module.exports = {
           "in your project's truffle-config.js."
       }
     ],
-    allowedGlobalOptions: ["network", "config"]
+    // The 'network' option is excluded from here and added manually above as a workaround to combine its usage
+    // instructions with url to show that they are mutually exclusive.
```

Comment:
What's the problem, and how does this workaround function?

Reply:
If `network` option is included in the global options, it is appended to the usage section in the end and we cannot show it as mutually exclusive with the `url` option.